app/views/devise/registrations/new.html.erb

```diff
@@ -1,24 +1,24 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= devise_error_messages! %>
 
-  <div class="subtitle">Start telling your stories and maintaining your health in a safe, private space</div>
+  <div class="subtitle"><%= t('devise.registrations.new.subtitle') %></div>
   <div class="spacer"></div>
 
   <div class="center">
-    <div class="field"><div class="label"><%= f.label 'Name' %></div>
+    <div class="field"><div class="label"><%= f.label t('common.name') %></div>
     <%= f.text_field :name %></div>
 
-    <div class="field"><div class="label"><%= f.label 'A few details' %></div>
-    <%= f.text_field :location, class: 'spacing', placeholder: 'Location' %>
-    <%= f.email_field :email, placeholder: 'Email' %></div>
+    <div class="field"><div class="label"><%= f.label t('devise.registrations.new.label') %></div>
+    <%= f.text_field :location, class: 'spacing', placeholder: t('common.location') %>
+    <%= f.email_field :email, placeholder: t('email') %></div>
```

Comment:
I think this one should be `t('common.email')` since the top-level email is `join.ifme@gmail.com` :upside_down_face: